examples/chef/common/clusters/audio-output/AudioOutputManager.h

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/audio-output-server/audio-output-server.h>
+#include <vector>
+
+class AudioOutputManager : public chip::app::Clusters::AudioOutput::Delegate
+{
+    using OutputInfoType = chip::app::Clusters::AudioOutput::Structs::OutputInfoStruct::Type;
+
+public:
+    AudioOutputManager();
+
+    uint8_t HandleGetCurrentOutput() override;
+    CHIP_ERROR HandleGetOutputList(chip::app::AttributeValueEncoder & aEncoder) override;
+    bool HandleRenameOutput(const uint8_t & index, const chip::CharSpan & name) override;
+    bool HandleSelectOutput(const uint8_t & index) override;
+    char * Data(uint8_t index) { return mCharDataBuffer[index]; }
+
+protected:
+    uint8_t mCurrentOutput = 1;
+    std::vector<OutputInfoType> mOutputs;
+    // Magic numbers are here on purpose, please allocate memory
+    static constexpr size_t mBufMax = 32;
```

Comment:
please give it a name for what this is. is this mNameLengthMax or something else? Otherwise comments should explain possible usage.

Reply:
Renamed to `mNameLenMax`